spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -301,6 +343,14 @@ def set_pdb_execute_events(self):
         self.call_kernel(interrupt=True).set_pdb_execute_events(
             CONF.get('run', 'pdb_execute_events', True))
 
+    def set_pdb_use_exclamation_mark(self):
+        """Set pdb_use_exclamation_mark into a debugging session"""
+        self.call_kernel(interrupt=True).set_pdb_use_exclamation_mark(
+            self.is_pdb_using_exclamantion_mark())
+
+    def is_pdb_using_exclamantion_mark(self):
+        return CONF.get('run', 'pdb_use_exclamation_mark', False)
```

Comment:
There's no need to pass `False` here as a default value because `pdb_use_exclamation_mark` is declared in our config system.